pkg/vm/client.go

```diff
@@ -43,7 +45,7 @@ func (c *Client) Shutdown(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	response, err := c.client.Do(request)
+	response, err := c.client.Do(&retryablehttp.Request{Request: request})
```

Comment:
I see from the docs we can still use the standard client with retryable client. if so, no need to wrap the request like the above
https://github.com/hashicorp/go-retryablehttp#getting-a-stdlib-httpclient-with-retries
so if we needed to change the client to something else no need to change those lines again